Src/Plugins/Security/SEC1001.SecurePlaintextSecrets.json

```diff
@@ -30,7 +30,7 @@
         },
         {
           "SubId": "GoogleApiKey",
-          "ContentsRegex": "AIza(?i)[0-9a-z-_]{35}",
+          "ContentsRegex": "(?<fingerprint>AIza(?i)[0-9a-z-_]{35})",
```

Comment:
>fingerprint [](start = 31, length = 11)

you actually don't need to add the fingerprint group when the entire match is the fingerprint: the tool will just use the primary match in any case. removing <fingerprint> here will result in one fewer regex match